_rules/aria-state-or-property-not-prohibited-yyw7k3.md

```diff
@@ -0,0 +1,160 @@
+
+## Assumptions
+
+There are no assumptions.
+
+## Accessibility Support
+
+Implementation of [Presentational Roles Conflict Resolution][] varies from one browser or assistive technology to another. Depending on this, some elements can have a [semantic role][] of `none` and their attributes fail this rule with some technologies but users of other technology would not experience any accessibility issue.
+
+## Background
+
+The presence of prohibited ARIA attributes is often the result of a developer using an incorrect role, or a misunderstanding of the attribute. These attributes are ignored by browsers and other assistive technologies. This often means that a state or property which should exist is missing. This can cause issues under success criterion [1.3.1 Info and Relationships][sc131] or [4.1.2 Name, Rule Value][sc412].
+
+Assessing the value of the attribute is out of scope for this rule.
+
+### Related rules
+
+- [ARIA state or property has valid value](https://www.w3.org/WAI/standards-guidelines/act/rules/6a7281/proposed/)
+- [ARIA state or property is permitted](https://www.w3.org/WAI/standards-guidelines/act/rules/5c01ea/proposed/)
+
+### Bibliography
+
+- [Understanding Success Criterion 4.1.1: Parsing](https://www.w3.org/WAI/WCAG21/Understanding/parsing.html)
+- [Understanding Success Criterion 4.1.2: Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html)
+- [WAI-ARIA 1.2, Supported States and Properties](https://www.w3.org/TR/wai-aria-1.2/#states_and_properties)
+- [WAI-ARIA 1.2, Global States and Properties](https://www.w3.org/TR/wai-aria-1.2/#global_states)
+- [ARIA5: Using WAI-ARIA state and property attributes to expose the state of a user interface component](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA5)
+- [Document conformance requirements for use of ARIA attributes in HTML](https://www.w3.org/TR/html-aria/#docconformance)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+The `aria-pressed` [state][] is [supported][] for the `button` role, which is the [implicit role][] for `button` elements.
+
+```html
+<button aria-pressed="false">My button</button>
+```
+
+#### Passed Example 2
+
+The `aria-pressed` [state][] is [supported][] for the `button` role, which is the [explicit role][] of this `div` element.
+
+```html
+<div role="button" aria-pressed="false">My button</div>
+```
+
+#### Passed Example 3
+
+The `aria-labelledby` [property][] is [supported][] for the `textbox` role, which is the [implicit role][] for and `input` element with `type="text"`.
+
+```html
+<input name="searchtxt" type="text" aria-labelledby="searchbtn" />
+<input name="searchbtn" id="searchbtn" type="submit" value="Search" />
+```
+
+### Failed
+
+#### Failed Example 1
+
+The `aria-label` property is [prohibited](https://www.w3.org/TR/wai-aria-1.2/#prohibitedattributes) for an element with a generic role.
+
+```html
+<div aria-label="Bananas"></div>
```

Comment:
I'm not sure this element is actually in the accessibility tree. ARIA 1.3 draft contradicts itself on that:

> https://w3c.github.io/aria/#scope
> user agents MUST ignore any elements with the role generic or none.

> https://w3c.github.io/aria/#generic
> However, unlike elements with role presentation, generic elements are exposed in accessibility APIs

Reply:
I could leave or delete this example, I think we could point to this sentence and say that it should be in the tree. 

> However, unlike elements with role presentation, user agents expose generic elements in [accessibility APIs](https://w3c.github.io/aria/#dfn-accessibility-api) when permitted accessibility attributes have been specified.